tools/policy-gen/protoc-gen-kumapolicy/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/hoisie/mustache"
```

Comment:
Yes, I was just thinking if we already use `mustache` somewhere in the project we want to be consistent. But I guess it's safe to use go templating here since it's not exposed to the user

Reply:
Yeah especially when we want to stop using `mustache`: https://github.com/kumahq/kuma/issues/2413